src/components/generics/vacation-test-forms/employee-vacation-requests.tsx

```diff
@@ -293,12 +326,12 @@ const RenderEmployeeVacationRequests = ({ persons }: Props) => {
    * @param id
    * @returns foundPerson.firstName and foundPerson.lastName
    */
-  const handlePersonNames = (id: number) => {
-    const foundPerson = persons.find(p => p.id === id);
+  const handlePersonNames = (id: string) => {
+    const foundPerson = persons.find(p => p.keycloakId === id);
     if (foundPerson) {
       return `${foundPerson.firstName} ${foundPerson.lastName}`;
     }
-    return "";
+    return "null";
```

Comment:
Why are we returning null as a string here? It might be clearer to return null and then return the desired 'error string' in the case of no foundPerson e.g.
 <StyledTableCell>{ handlePersonNames(request.personId!) || "null" }</StyledTableCell>

Is "null" the most appropriate message here. Should it be localized using the strings package?

Reply:
Made the function return a localized string and the function call return a null if personId not provided. Also added two new error strings to the strings package. Are the new error messages appropriate?

employee-vacation-requests.tsx
```suggestion
    const handlePersonNames = (id: string | null) => {
      if (id === null) {
        return `${strings.errorHandling.personIdUndefined}`;
      }
      const foundPerson = persons.find(p => p.keycloakId === id);
      if (foundPerson) {
        return `${foundPerson.firstName} ${foundPerson.lastName}`;
      }
      return `${strings.errorHandling.nameNotFound}`;
    };
```
en.json
```suggestion
      "errorHandling": {
        ...
        "nameNotFound": "Name not found",
        "personIdUndefined": "Person ID undefined"
      },
```